public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -894,21 +872,13 @@ const ContactPointsHoverDetails: FC<ContactPointDetailsProps> = ({
       }
       key={uniqueId()}
       content={
-        <Stack direction="row" gap={0.5}>
-          {/* use "label" to indicate how many of that type we have in the contact point */}
-          {Object.entries(groupedIntegrations).map(([type, integrations]) => (
-            <Label
-              key={uniqueId()}
-              label={integrations.length > 1 ? integrations.length : undefined}
-              icon={INTEGRATION_ICONS[type]}
-              value={upperFirst(type)}
-            />
-          ))}
+        <Stack justifyContent="center">
```

Comment:
It looks a bit odd being centered, I'll play around with the card design a bit to see what could work. What if we made the header smaller too?

<img width="213" alt="image" src="https://github.com/user-attachments/assets/9fbc8982-c6fc-4030-9fcb-b5552d8181f6">